encryption/data_converter_test.go

```diff
@@ -0,0 +1,36 @@
+package encryption_test
+
+import (
+	"testing"
+
+	"github.com/canonical/temporal-lib-go/encryption"
+
+	"github.com/stretchr/testify/require"
```

Comment:
This is a half-joke 😄, but let's promote our Canonical colleagues, not other engineers! What about using `"github.com/go-quicktest/qt"`?

```go
import "github.com/go-quicktest/qt"

func TestDataConverter(t *testing.T) {
	defaultDc := converter.GetDefaultDataConverter()

	cryptDc, err := encryption.NewEncryptionDataConverter(
		converter.GetDefaultDataConverter(),
		encryption.EncryptionOptions{
			Key: "HLCeMJLLiyLrUOukdThNgRfyraIXZk918rtp5VX/uwI=",
		},
	)
	qt.Assert(t, qt.IsNil(err))

	defaultPayloads, err := defaultDc.ToPayloads("Testing")
	qt.Assert(t, qt.IsNil(err))

	encryptedPayloads, err := cryptDc.ToPayloads("Testing")
	qt.Assert(t, qt.IsNil(err))

	qt.Assert(t, qt.Not(qt.Equals(
		defaultPayloads.Payloads[0].GetData(),
		encryptedPayloads.Payloads[0].GetData(),
	)))

	var result string
	err = cryptDc.FromPayloads(encryptedPayloads, &result)
	qt.Assert(t, qt.IsNil(err))
	qt.Assert(t, qt.Equals("Testing", result))
}
```

When this library gets imported, a developer will get this package and consider using this testing library.

Reply:
Thanks for the suggestion and code :smile: I've updated the test to use `qt` instead as per your recommendation.